### ISSUE TITLE
application cannot start because of contradictory default settings

### ISSUE THREAD

**Issue by HendrikSchrieber** [gabe565/charts#267]
### Chart Name

vaultwarden

### Chart Version

0.9.1

### Bug Description

When trying to run the vaultwarden chart with default settings it is unable to run because  in default settings http-port is set to port 80 but podSecurityContext is set to run as non-root user. The vaultwarden rocket server will fail to start with a permission denied error and pod will go into a restart-loop because ports up to 1024 can only be bound by root user. In favor of a secure setup the default http-port should be changed to a port > 1024. See https://github.com/dani-garcia/vaultwarden/issues/411#issuecomment-466673242.

### Relevant log output

```shell
[2023-07-28 08:49:37.872][vaultwarden::api::notifications][INFO] Starting WebSockets server on 0.0.0.0:3012
Error: Rocket.
[CAUSE] Bind(
    Os {
        code: 13,
        kind: PermissionDenied,
        message: "Permission denied",
    },
)
```

**Comment by gabe565** [comment]
@HendrikSchrieber It's interesting that this is causing an issue for you. In my personal Kubernetes cluster, `net.ipv4.ip_unprivileged_port_start` always seems to be set to 0, so Vaultwarden is able to bind to 80 without issue. That being said, it would be safer to bind to a non-privileged port. I'll go ahead and update the chart!

**Comment by gabe565** [comment]
Also, a quick note. My Vaultwarden chart will automatically set the `ROCKET_PORT` env to match the value of `service.main.ports.http.port`. So, while I'm researching and releasing this update, you can go ahead and set something like this in your `values.yaml`:
```yaml
service:
  main:
    ports:
      http:
        port: 8080
```

**Comment by gabe565** [comment]
@HendrikSchrieber Ok I just released `v0.10.0` which removes the WebSocket port, and sets the default HTTP port to `8080`. Can you try again with that version?

**Comment by HendrikSchrieber** [comment]
Works for me, thanks for the fix :+1:
